package/packageConfig.js

```diff
@@ -60,6 +70,23 @@ async function addDependency(dep) {
   writePackageConfig(config);
 }
 
+function getNpmDependencies() {
+  return getPackageConfig().npm_dependencies;
+}
+
+function fetchNpmDependencies(destination) {
+  process.chdir(destination);
+  return new Promise((resolve, reject) =>
+    exec("npm install", err => {
```

Comment:
I suggest using a `spawn` instead of an exec here. The reason is that with `spawn` we can subscribe to the `message` event and get real time console output from the process. `exec` returns the whole output in a single batch.

Here's an example:
```js
const installNpmDependencies = cwd =>
  new Promise((resolve, reject) => {
    const install = spawn("npm", ["install"], { cwd });
    install.on("close", resolve);
    install.on("error", reject);
  });
```

Notice that in this case I am totally ignoring the `message` event, but implementing this way makes the function pretty extensible and debuggable.

Reply:
Fixed in c906a1a26dbc607889104c13a7220623abf03d94